pkg/devfile/generator/generators_test.go

```diff
@@ -436,8 +446,8 @@ func TestGetVolumesAndVolumeMounts(t *testing.T) {
 			}
 
 			pvcVols, err := GetVolumesAndVolumeMounts(devObj, volumeParams, common.DevfileOptions{})
-			if tt.wantErr == (err == nil) {
-				t.Errorf("TestGetVolumesAndVolumeMounts() error = %v, wantErr %v", err, tt.wantErr)
+			if (tt.wantErr == nil) != (err == nil) {
```

Comment:
can we keep one or the other, in the above test it was 

`if (err != nil) != (tt.wantErr != nil)`. Otherwise I am always thrown back and have to recalc it in my head 😛

Reply:
updated